jobs/src/main/scala/dcos/metronome/utils/glue/MarathonImplicits.scala

```diff
@@ -68,11 +68,8 @@ object MarathonImplicits {
   implicit class JobSpecToContainer(val jobSpec: JobSpec) extends AnyVal {
     def toContainer: Option[Container] = {
       jobSpec.run.docker match {
-        case Some(dockerSpec) => Some(Container(
-          `type` = mesos.Protos.ContainerInfo.Type.DOCKER,
-          docker = Some(Container.Docker(
-            image = dockerSpec.image
-          )),
+        case Some(dockerSpec) => Some(Container.Docker(
+          image = dockerSpec.image,
```

Comment:
Does 1.3.x not have support for UCR?

Reply:
It does. So far, Metronome only supports Docker via its API, but adding UCR as an option sould be a good next step after bumping.